benchmark/cli_args.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef CLI_ARGS_HPP
```

Comment:
Could this file be named more descriptively - for example `benchmark_cli_arg_defs.hpp`. When reading other code that includes it, it's not clear whether or not it's just definitions for general CLI argument parsing, or for benchmarks specifically.

Reply:
Done